contracts/tests/sbtc-token.test.ts

```diff
@@ -101,6 +101,76 @@ describe("sBTC token contract", () => {
       expect(receipt2.value).toEqual(999n);
     });
 
+    test("Mint & transfer multiple sbtc token", () => {
+      const { burnHeight, burnHash } = getCurrentBurnInfo();
+      const receipt = txOk(
+        deposit.completeDepositWrapper({
+          txid: new Uint8Array(32).fill(0),
+          voutIndex: 0,
+          amount: 1000n,
+          recipient: alice,
+          burnHash,
+          burnHeight,
+          sweepTxid: new Uint8Array(32).fill(1),
+        }),
+        deployer
+      );
+      const printEvents = filterEvents(
+        receipt.events,
+        CoreNodeEventType.ContractEvent
+      );
+      const [print] = printEvents;
+      const printData = cvToValue<{
+        topic: string;
+        txid: string;
+        voutIndex: bigint;
+        amount: bigint;
+      }>(print.data.value);
+      expect(printData).toStrictEqual({
+        topic: "completed-deposit",
+        bitcoinTxid: new Uint8Array(32).fill(0),
+        outputIndex: 0n,
+        amount: 1000n,
+        burnHash,
+        burnHeight: BigInt(burnHeight),
+        sweepTxid: new Uint8Array(32).fill(1),
+      });
+      const receipt1 = txOk(
+        token.transferMany({
```

Comment:
Add a test with a contract as contract-caller, so that it is shown that sBTC is a "safe transferable" token.

Reply:
Was exploring this today & I think the best way to do this is probably not to create an actual contract but to create one at runtime when tests are executed. Have you done this before in Clarinet?

Reply:
I added a test contract 'sbtc-token-test' that tests a contract call properly call the main function in this PR (transfer-many).